lib/rdf.js

```diff
@@ -127,6 +127,28 @@ function createRDF(manifest) {
     });
   }
 
+  if (manifest.locales) {
+    for (var locale in manifest.locales){
+      var l10n = manifest.locales[locale];
+      var l10nDescription = {
+        "em:locale": locale,
+        "em:name": l10n.title || manifest.title || manifest.name || "Untitled",
+        "em:description": l10n.description || manifest.description || undefined,
```

Comment:
I kept that just because there were ` || undefined` in line 52, and line 54 or line 55. I wonder if that 3 can be removed? or I'd just keep it there